chapters/appendix-X-custom-license-namespaces.md

```diff
@@ -0,0 +1,48 @@
+
+The SPDX specification also defines a mechanism for creators of SPDX documents to define custom license texts and identifiers. This enables SPDX documents to refer to licenses that might never be included on the SPDX License List, for a variety of reasons (whether due to limited breadth of use, failure to meet the [license inclusion principles](https://github.com/spdx/license-list-XML/blob/master/DOCS/license-inclusion-principles.md), or otherwise).
+
+These custom license identifiers are prefixed with `LicenseRef-` as defined in [Appendix IV of the SPDX 2.2 Specification](./appendix-IV-SPDX-license-expressions.md). The `LicenseRef-` identifier format was originally assumed to be used in the context of an SPDX document, so that the license text for that identifier would be specified in a corresponding [Other Licensing Information](./6-other-licensing-information-detected.md) section.
+
+Subsequently, interest has been expressed in being able to use stable identifiers for custom licenses outside the context of a single SPDX document. For example, developers might wish to insert [SPDX short-form license identifiers](https://spdx.org/ids) into their source code, for licenses that are not on the License List. Additionally, tools may want to interchange information about custom licenses without having to assume that they are both using the same internal format to generate `LicenseRef-` identifiers, or without having to compare defined text from an Other License Information section of an SPDX document to confirm that they are in fact the same text.
+
+## Brief overview
+
+This appendix defines a format and mechanism for using the existing `LicenseRef-` format in a manner that enables stable identifier usage. Under this system:
+
+1. each license "namespace" consists of a standard order DNS name (such as `example.com`) that is under the control of the namespace maintainer's organization;
+2. the namespace maintainer creates an SPDX document defining license texts and corresponding identifiers with the DNS name included, as described below, and publishes it publicly at a URL under the namespace domain;
+3. the namespace and the SPDX document URL is registered with the SPDX project; and
+4. identifiers in the format described below can then be used by anyone, as long as they are defined in the SPDX document available at the registered URL.
+
+## Format
+
+For purposes of this appendix, a valid "namespace" is a standard order DNS name, such as `example.com`. Each custom license identifier under this namespace consists of the following, in order:
```

Comment:
Update to also include org name as a second format option, per discussion on tech team call 2020-02-25

Reply:
added in 5976c1d